src/legacy/core_plugins/tests_bundle/index.js

```diff
@@ -52,10 +52,7 @@ export default (kibana) => {
           }
         } = kbnServer;
 
-        const testGlobs = [
-          'src/legacy/ui/public/**/*.js',
-          '!src/legacy/ui/public/flot-charts/**/*',
-        ];
+        const testGlobs = [];
```

Comment:
On master there are 500+ browser tests, but in this PR we're getting less than 100 https://kibana-ci.elastic.co/job/elastic+kibana+master/118/JOB=kibana-intake,node=linux-immutable/testReport/Browser%20Unit%20Tests/

I assume it's caused by this, we need to figure out how to select the right number of files in the types of tests.

Reply:
ahh, yeah I recall we discussed this.  Darn.  On it!